### ISSUE TITLE
Upgrade neighbourhood shapes for Rome

### ISSUE THREAD

**Issue by nvkelso** [whosonfirst-data/whosonfirst-data#420]
### tl;dr

Status:
- [x] Propose **sources**
- [x] Establish **concordances** between WOF and source
- [ ] **Upgrade / add** WOF shapes
- [ ] **Add new or missing** features to WOF
- [ ] **Deprecate / supersede** WOF features
- [ ] **Draft review**
- [ ] **Draft revisions** based on review comments
- [ ] **Final review**
- [ ] **Final revisions** based on review comments
- [ ] **Import** into Who's On First
### Details

Sourcing:
- Permissive license?
- Add new src to Who's On First

Upgrade / add:
- `macrohood`
- `neighbourhood`
- `microhood`

Add new or missing features to WOF
- `macrohood`
- `neighbourhood`
- `microhood`

Deprecate / supersede:
- `macrohood`
- `neighbourhood`
- `microhood`

**Comment by nvkelso** [comment]
**Possible sources:**

- https://en.wikipedia.org/wiki/Administrative_subdivision_of_Rome

**Comment by justinelliotmeyers** [comment]
https://www.comune.roma.it/TERRITORIO/geoserver/ows?service=WFS&version=1.0.0&request=GetFeature&typeName=toponomastica:SuddivisioniToponomastica&SRSNAME=EPSG:6708&outputFormat=SHAPE-ZIP

https://www.comune.roma.it/TERRITORIO/geoserver/ows?service=WFS&version=1.0.0&request=GetFeature&typeName=statistica:ZoneUrbanistiche&SRSNAME=EPSG:6708&outputFormat=SHAPE-ZIP


@nvkelso  @stepps00 I just emailed you, then found these, need to determine what layer to use and most importantly, copyright.

Source:
https://www.comune.roma.it/TERRITORIO/nic-gwt/

**Comment by stepps00** [comment]
This is great, thanks @justinelliotmeyers. I dug into both datasets, it's a bit confusing trying to understand the difference between the two divisions.

It looks like the Suddivisioni Toponomastica data represent "historical subdivisions", made up of 22 "rioni", 35 "quartieri", 6 "suburbi", and 53 "zones."

While the Zone Urbanistiche are defined as: "the urban subdivision represented by the 155 urban zones into which the Municipalities of Rome are divided. They were established in 1977 for statistical and territorial planning and management purposes, according to criteria of homogeneity from the urbanistic point of view. The boundaries are identified along the more or less marked continuity solutions of the urban fabric." {Translated}

Source: https://it.wikipedia.org/wiki/Suddivisioni_di_Roma

That being said, it does look like other mapping platforms use the Suddivisioni Toponomastica geometries and labels for neighbourhoods in Rome, which is probably what I'd propose for WOF. Before import, I'd want to dig into some of the geometry differences between these datasets though (geometries for a place seem to differ between them).

Lastly, maybe this general Copyright page is helpful? https://www.comune.roma.it/web/it/note-legali.page

**Comment by justinelliotmeyers** [comment]
http://websit.cittametropolitanaroma.it/Download.aspx

has good copyright if we need to use a different source - split out by similar source, did not check actual data

**Comment by stepps00** [comment]
<img width="1193" alt="Screenshot 2023-01-26 at 1 13 05 PM" src="https://user-images.githubusercontent.com/18567700/214951708-f1a11d4c-2620-4819-a1ab-6d2516751366.png">

Prepping some new neighbourhood data in Rome, loosely based on the [rioni](https://en.wikipedia.org/wiki/Rioni_of_Rome) and [quartieri](https://en.wikipedia.org/wiki/Quarters_of_Rome) in the city. Concordances have been established, and Italian names have been appended to each feature. I'll look to merge this into the admin-it repo shortly and update this issue when a PR is ready for review.
